presto-hive/src/main/java/com/facebook/presto/hive/ZeroRowFileCreator.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.hive.metastore.StorageFormat;
+import com.facebook.presto.spi.ConnectorSession;
+import org.apache.hadoop.fs.Path;
+
+import java.util.List;
+import java.util.Properties;
+
+public interface ZeroRowFileCreator
+{
+    void createFiles(ConnectorSession session, HdfsEnvironment.HdfsContext hdfsContext, Path destinationDirectory,
```

Comment:
nit: We don't use this style of declaring parameters. We either put all the parameters on the same line, or every parameter on a new line

```
void createFiles(
  ConnectorSession session,
  HdfsContext hdfsContext,
  Path destinationDirectory,
  List<String> fileNames,
  StorageFormat format,
  HiveCompressionCodec compressionCodec, 
  Properties schema);
```